documentation/specs/resource-management.md

```diff
@@ -0,0 +1,45 @@
+# Managing tools with their own parallelism in MSBuild
+
+MSBuild supports building projects in parallel using multiple processes. Most users opt into `NUM_PROCS` parallelism at the MSBuild layer.
```

Comment:
The string `NUM_PROCS` does not exist in MSBuild. Maybe write a paragraph or two detailing where it comes from.

Reply:
```suggestion
MSBuild supports building projects in parallel using multiple processes. Most users opt into `Environment.ProcessorCount` parallelism at the MSBuild layer.
```